smartautoclicker/src/main/res/values-zh/strings.xml

```diff
@@ -0,0 +1,147 @@
+    <string name="dialog_action_copy_pause_details">持续 %1$s</string>
+    <string name="dialog_action_copy_header_event">来自该情景下的所有动作</string>
+    <string name="dialog_action_copy_header_all">来自所有情景下的所有动作</string>
+    <string name="dialog_action_copy_search">通过动作名寻找</string>
+
+    <string name="dialog_condition_title">条件</string>
+    <string name="dialog_condition_name_title">名字</string>
+    <string name="dialog_condition_detection_details_title">检测详情</string>
+    <string name="dialog_condition_delete">删除</string>
+    <string name="dialog_condition_at">在位置 [%1$d,%2$d] [%3$d,%4$d]</string>
+    <string name="dialog_condition_type_title">检测类型</string>
+    <string name="dialog_condition_type_whole_screen">屏幕任意位置</string>
+    <string name="dialog_condition_type_exact_position">在相同位置</string>
+    <string name="dialog_condition_threshold_title">误差允许程度</string>
+    <string name="dialog_condition_threshold_value">误差: %1$d %%</string>
+    <string name="dialog_condition_should_be_detected">看到</string>
+    <string name="dialog_condition_should_not_be_detected">没看到</string>
+    <string name="dialog_condition_error">条件图像未能找到</string>
+    <string name="dialog_condition_new_title">新条件</string>
+    <string name="dialog_condition_new_create">创建</string>
+    <string name="dialog_condition_new_copy">从其他条件拷贝</string>
+    <string name="dialog_condition_copy_empty">不能拷贝，现在没有其他条件。</string>
+    <string name="dialog_condition_copy_threshold">%1$d%%</string>
+
+    <string name="toast_configure_single_click">触摸到点击的位置</string>
+    <string name="toast_configure_swipe_from">触摸滑动开始位置</string>
+    <string name="toast_configure_swipe_to">触摸滑动结束位置</string>
+
+    <string name="accessibility_service_description">该应用需要无障碍服务来实现自动点击。
+        一旦条件被检测到，便会执行由你创建的，位于其他应用上的点击列表。\n\n 该应用没有收集任何数据。 您可以自行访问我们的Github，代码是开源的!</string>
+
+    <string name="content_desc_add_scenario">创建一个新场景</string>
+    <string name="content_desc_image_condition">要被检测的图像</string>
+    <string name="content_desc_event_action_btn">编辑这个情景</string>
+    <string name="content_desc_delete_scenario">删除场景</string>
+    <string name="content_desc_play_pause_scenario">开始或者停止该场景</string>
+    <string name="content_desc_stop_clicker">停止自动点击器</string>
+    <string name="content_desc_open_event_list">打开情景列表</string>
+    <string name="content_desc_move_menu">移动悬浮窗菜单</string>
+    <string name="content_desc_confirm">确认</string>
+    <string name="content_desc_go_back">返回到对话框</string>
+    <string name="content_desc_overlay_state">悬浮窗权限状态</string>
+    <string name="content_desc_accessibility_state">无障碍服务权限状态</string>
+    <string name="content_desc_action">要被执行的动作</string>
+    <string name="content_desc_action_icon">这个图标标注动作类型</string>
+    <string name="content_desc_dialog_title_icon">对话框标题图标</string>
+
+    <string name="dialog_unit_ms" translatable="false">ms</string>
+    <string name="dialog_copy_title">拷贝来自</string>
```

Comment:
Recent changes have added a few new strings, if you can update your commit this would be great !